README.MD

```diff
@@ -68,12 +69,22 @@ You can set in the settings file `get remote artwork` and `create cache file`. T
 Right now you can only use the old library, so don't use this for now.
 
 ### The Spotify Way
-If you would like to key all of your Spotify playlists, go to the [MDRP website](https://mdrp.tech/fetchalbumart) (under construction, please be patient) and click `Regenerate Token`, then select either `Include Artists` (recommended) or `Exclude Artists` and then download the dat (may take up to 5 minutes if you have a ton of songs) and place it in your clientdata folder.
+If you would like to key all of your Spotify playlists, go to the [MDRP website](https://smaltin.dev/fetchalbumart) (under construction, please be patient) and click `Regenerate Token`, then select either `Include Artists` (recommended) or `Exclude Artists` and then download the dat (may take up to 5 minutes if you have a ton of songs) and place it in your clientdata folder.
 
 ## Changing MDRP Settings
 To change any MDRP settings, they will all be located in your `DiscordPresenceConfig.ini` file.\
 Data in this file is in `key=value` pairs so any line that does not have a `=` will be ignored.
 
+## Adding a custom player
+***Assuming the player outputs properly to WMC***, it can be added as a new player. Open `SupportedPlayers.dat` and following the schema below, fill out the information.
```

Comment:
Possibly explain here how to figure out if output happens, or reference how to find out?